app/views/batch_contexts/index.html.erb

```diff
@@ -1,7 +1,36 @@
 <div class="container">
-  <h2>Projects</h2>
+  <h2>Projects History</h2>
 
   <%= paginate @batch_contexts %>
-    <%= render @batch_contexts %>
-  <%= paginate @batch_contexts %>
+
+  <table class="table table-bordered">
+    <thead>
+      <tr class="table-secondary">
+        <th scope="col">Name</th>
+        <th scope="col">Creator</th>
+        <th scope="col">Content Structure</th>
+        <th scope="col">Staging Location</th>
+        <th scope="col">Processing Configuration</th>
+        <th scope="col">Using File Manifest</th>
+        <th scope="col"># Job Runs</th>
+        <th scope="col">Created At</th>
+      </tr>
+    </thead>
+    <tbody>
+    <% @batch_contexts.each do |batch_context| %>
+        <tr>
+          <td><%= link_to batch_context.project_name, batch_context %></td>
+          <td><%= batch_context.user.email %></td>
+          <td><%= batch_context.content_structure %></td>
+          <td><%= batch_context.staging_location %></td>
+          <td><%= batch_context.processing_configuration %></td>
+          <td><%= batch_context.using_file_manifest ? 'yes' : 'no' %></td>
+          <td><%= batch_context.job_runs.count %></td>
+          <td><%= batch_context.created_at.in_time_zone('Pacific Time (US & Canada)').to_formatted_s(:long) %></td>
+        </tr>
+    <% end %>
+   </tbody>
+  </table>
+
+<%= paginate @batch_contexts %>
 </div>
```

Comment:
instead of rendering the detail view over and over again, render it nicely as a table